packages/api-explorer/src/scenes/MethodScene/components/DocResponse.tsx

```diff
@@ -38,28 +40,33 @@ import {
 } from '@looker/components'
 import { IMethodResponse } from '@looker/sdk-codegen'
 
+import styled from 'styled-components'
+
 import { DocCode } from '../../../components'
 import { copyAndCleanResponse } from '../utils'
 
 interface DocResponseProps {
   responses: IMethodResponse[]
 }
+
+const StyledHeading = styled(Heading)``
```

Comment:
Not sure I underwant why this is here :)

Reply:
Removed, unneeded.